pose_evaluation/metrics/embedding_distance_metric.py

```diff
@@ -0,0 +1,104 @@
+from typing import Literal, Union, List
+import torch
+import torch.nn.functional as F
+import numpy as np
+
+from pose_evaluation.metrics.base_embedding_metric import EmbeddingMetric
+
+
+class EmbeddingDistanceMetric(EmbeddingMetric):
+    def __init__(self, kind: Literal["cosine", "l2"] = "cosine", device: Union[torch.device, str] = None):
```

Comment:
"euclidean" instead of "l2" i think is better

Reply:
agreed